src/containers/VehiclesAddForm/VehiclesAddForm.js

```diff
@@ -1,49 +1,158 @@
-import React from 'react';
+import React, {useState} from 'react';
 
 import {
-  Button,
-  TextField,
-  Select,
-  MenuItem,
-  InputLabel
+    Button,
+    TextField,
+    Select,
+    MenuItem,
+    InputLabel,
+    Container,
+    FormControl
 } from '@material-ui/core';
 
 import './style.css';
 
-function createData(index, vehicle) {
-    return { index, vehicle };
-}
 
-const rows = [
-    createData(1, 'Corsa'),
-    createData(2, 'Uno'),
-    createData(3, 'Gol'),
-    createData(4, 'Argo'),
-    createData(5, 'Fiesta'),
-];
-
-const VehiclesAddForm = () => (
-  <form className="vehiclesAddForm" noValidate autoComplete="off">
-    <div className="vehiclesAddForm--select">
-        <InputLabel id="standard-basic">Marca</InputLabel>
-        <Select id="standard-basic" label="Marca">
-            {rows.map((row) => {
-                return <MenuItem value={row.index}>{row.vehicle}</MenuItem>;
-            })}
-        </Select>
-    </div>
-    <div className="vehiclesAddForm--inputs">
-        <TextField id="standard-basic" label="Modelo" />
-        <TextField id="standard-basic" label="Ano" />
-        <TextField id="standard-basic" label="Valor" />
-    </div>
-    <Button variant="contained" color="primary">
-        Cadastrar
-    </Button>
-    <Button variant="contained" color="secondary">
-        Cancelar
-    </Button>
-  </form>
-)
+const VehiclesAddForm = ({toSend}) => {
+
+    function createData(index, vehicle) {
+        return {index, vehicle};
+    }
+
+    const rows = [
+        createData(1, 'Corsa'),
+        createData(2, 'Uno'),
+        createData(3, 'Gol'),
+        createData(4, 'Argo'),
+        createData(5, 'Fiesta'),
+    ];
+
+    const [brand, setBrand] = useState("");
+    const [model, setModel] = useState("");
+    const [year, setYear] = useState("");
+    const [value, setValue] = useState("");
+    const [errors, setErrors] = useState({
+        validator: {
+            buttonEnabled: false,
+            isValid: true,
+            text: ""
+        }
+    });
+
+    const handleMarcaSelectionChange = (event) => {
+        setBrand(event.target.value);
+    };
+
+    const handleModeloChange = (event) => {
+        setModel(event.target.value);
+    };
+
+    const handleYearChange = (event) => {
+        setYear(event.target.value);
+    };
+
+    const handleValueChange = (event) => {
+        setValue(event.target.value);
+    };
+
+    function isValueValid(value){
+        if(+value <= 0){
+            return {
+                buttonEnabled: false,
+                isValid: false,
+                text: 'Adicione um valor maior que 0',
+            }
+        } else {
+            return { buttonEnabled:true, isValid: true, text: "" }
+        }
```

Comment:
acho que não precisa do `else` aqui, já que se o value ser menor ou igual a 0 ele já não cairá no `if`.
```suggestion
        } 
          return { buttonEnabled:true, isValid: true, text: "" }
        }
```